src/shared/sam/debugger/goSamDebug.ts

```diff
@@ -163,7 +163,7 @@ interface InstallScript {
  * @returns Path for the debugger install script, undefined if we already built the binary
  */
 async function makeInstallScript(debuggerPath: string, isWindows: boolean): Promise<InstallScript | undefined> {
-    let script: string = ''
+    let script: string = isWindows ? '' : '#!/bin/sh\n'
```

Comment:
`#!/usr/bin/env sh` ?

Reply:
I did some reading and it seems like `#!/bin/sh` is preferred over `#!/usr/bin/env sh` for `sh`. Plus this is the same behavior as POSIX spawn which is what we want.

For other shells you'd want to use `env`